src/config/chis-tg/config.json

```diff
@@ -10,7 +10,91 @@
       "domain": "moh-togo-uat.dev.medicmobile.org"
     }
   ],
-  "contact_types": [  
+  "contact_types": [
+    {
+      "name": "cg70_supervision",
+      "friendly": "Supervision Area",
+      "contact_type": "cg72_supervision",
+      "user_role": ["supervisor"],
+      "username_from_place": true,
+      "deactivate_users_on_replace": true,
+      "hierarchy": [
+        {
+          "friendly_name": "Sanitary Training",
+          "property_name": "sanitary_training",
+          "contact_type": "cf60_sanitary",
+          "type": "name",
+          "required": true,
+          "level": 1
+        }
+      ],
+      "replacement_property": {
+        "friendly_name": "Affected Supervision Area",
+        "property_name": "replacement",
+        "type": "string",
+        "parameter": ["\\(Supervision Area Name\\)", "\\(SUPERVISION AREA NAME\\)"],
+        "required": true
+      },
+      "place_properties": [
+        {
+          "friendly_name": "Supervision Area Name",
+          "property_name": "supervision_area",
+          "type": "string",
```

Comment:
In CHT contact forms, this appears to be autogenerated as the Supervisor contact's name. Should we do the same (`{{ contact.first }} {{ contact.last }} ({{ lineage.sanitary_training }})`), or do we really want to prompt for a unique area name here?

Reply:
Nice!